### ISSUE TITLE
Can't use qr_code_url. Got an exception.

### ISSUE THREAD

**Issue by Huluti** [endroid/qr-code-bundle#42]
Hi!

I try to show a QR Code in Twig like this:

```
{% set qrcode = qr_code_url(shorten_url) %}
<img src="{{ qrcode }}" />
```
However, I get this error:

`An exception has been thrown during the rendering of a template ("Parameter "data" for route "qr_code_generate" must match "[^/]++" ("https://localhost:8000/yvNRQ" given) to generate a corresponding URL.").`

The url is a string : "https://localhost:8000/yvNRQ"

It doesn't even work with manually passing any url like:
`<img src="{{ qr_code_url('https://github.com/endroid/qr-code-bundle/issues/42') }}" />`

**Comment by endroid** [comment]
Hi @Huluti QR code URLs indeed do not allow "/" characters, like the error message says. If you want to put a URL inside the QR /generate/{data} URL provided by the bundle you can try the following.

1. Override the routing requirements of the QR code generate URL provided by the bundle so you can pass a URL (read [this page](https://symfony.com/doc/4.2/routing/requirements.html) on how to do that)
2. Use the qr_code_data_uri method to put the qr code data directly in an image src in your template instead of working via the /generate/{data} URL.

Good luck :)

**Comment by tacman** [comment]
I used this bundle a few years ago, and I remember it working for urls.  

Are you saying it's not really a url, but rather just a domain?  It seems to work with 'example.com', but not https://example.com.

**Comment by endroid** [comment]
@tacman The v4 qr_code_url did not handle URLs correctly. This should be fixed in 4.0.4.
